pywbem_mock/_wbemconnection_mock.py

```diff
@@ -2816,13 +2818,13 @@ def _pull_response(self, namespace, req_type, **params):
             context_data = self.enumeration_contexts[context_id]
         except KeyError:
             raise CIMError(CIM_ERR_INVALID_ENUMERATION_CONTEXT,
-                           'EnumerationContext %s not found in mock server '
-                           'contexts.' % context_id)
+                           'EnumerationContext %r not found in mock server '
+                           'Enumeration contexts.' % context_id)
```

Comment:
Use lower case for "enumeration contexts".

Reply:
DONE